js/src/FirstRun/firstRun.js

```diff
@@ -16,353 +16,61 @@
-            key='next'
-            label={ BUTTON_LABEL_NEXT }
-            onClick={ this.onNext }
-          />
-        );
-
-      case 1:
-        return (
+        buttons={
           <Button
             disabled={ !hasAcceptedTnc }
-            icon={ <NextIcon /> }
-            key='next'
-            label={ BUTTON_LABEL_NEXT }
-            onClick={ this.onNext }
-          />
-        );
-
-      case 2:
-        const buttons = [
-          <Button
-            disabled={ !canCreate }
-            icon={ <NextIcon /> }
-            key='next'
-            label={ BUTTON_LABEL_NEXT }
-            onClick={ this.onNext }
+            icon={ <DoneIcon /> }
+            key='accept'
+            label='Close'
+            onClick={ this.store.close }
           />
-        ];
-
-        if (hasAccounts) {
-          buttons.unshift(
-            <Button
-              icon={ <NextIcon /> }
-              key='skip'
-              label={
-                <FormattedMessage
-                  id='firstRun.button.skip'
-                  defaultMessage='Skip'
-                />
-              }
-              onClick={ this.skipAccountCreation }
-            />
-          );
         }
-        return buttons;
-
-      case 3:
-        return [
-          <Button
-            icon={ <PrintIcon /> }
-            key='print'
-            label={
-              <FormattedMessage
-                id='firstRun.button.print'
-                defaultMessage='Print Phrase'
-              />
-            }
-            onClick={ this.printPhrase }
-          />,
-          <Button
-            disabled={ !!phraseBackedUpError }
-            icon={ <NextIcon /> }
-            key='next'
-            label={ BUTTON_LABEL_NEXT }
-            onClick={ this.onConfirmPhraseBackup }
-          />
-        ];
-
-      case 4:
-        return [
-          <Button
-            icon={ <ReplayIcon /> }
-            key='restart'
-            label={
-              <FormattedMessage
-                id='firstRun.button.restart'
-                defaultMessage='Start Over'
-              />
-            }
-            onClick={ this.onStartOver }
-          />,
-          <Button
-            icon={ <CheckIcon /> }
-            key='create'
-            label={
-              <FormattedMessage
-                id='firstRun.button.create'
-                defaultMessage='Create'
-              />
-            }
-            onClick={ this.onCreate }
-          />
-        ];
-
-      case 5:
-        return (
-          <Button
-            icon={ <DoneIcon /> }
-            key='close'
-            label={
-              <FormattedMessage
-                id='firstRun.button.close'
-                defaultMessage='Close'
-              />
-            }
-            onClick={ this.onClose }
+        hideClose
+        title={
+          <FormattedMessage
+            id='firstRun.title.termsOnly'
+            defaultMessage='Terms & Conditions'
```

Comment:
VSCode shows this & symbol as red, i.e. should be replaced be &amp;. Not sure though, we can leave it like this.

Reply:
Easier to update and be 100% sure.